src/assets/component-properties.json

```diff
@@ -0,0 +1,5 @@
+{
+  "player": {
+    
+  }
+}
```

Comment:
The locally stored global default file. If any feature details exist here, this version of details will be used instead of the one present in the incoming plio config

Reply:
Default will have those details which we want to be persistent across all the plios/components. Till now we have not decided what will be visible in all the plios. 
Let's say we decide we want to finalize the progress bar with some specific properties, so we will put that information here and it will be applied to all the plios